src/content/utils.ts

```diff
@@ -127,6 +126,16 @@ export async function extractSection(element: Element) {
   // Splitting title and section details into parts
   const titleParts = title.split(' - ');
 
+  // Concatenate section details back together if it got split 
+  if (titleParts.length > 2) {
```

Comment:
Can we change this to just split on the first occurrence of " - " instead?  This way we don't have to go and recombine it and I think we can even remove the check on length > 2.